jsk_panda_robot/jsk_panda.rosinstall

```diff
@@ -0,0 +1,20 @@
+# This file is for work space in pand PC.
+# Currently .rosintall is shared with user PC's one.
+
+
+# Wait until https://github.com/jsk-ros-pkg/jsk_robot/pull/1343 to be merged
+- git:
+    local-name: jsk-ros-pkg/jsk_robot
+    uri: https://github.com/ykawamura96/jsk_robot.git
+    version: add_panda_robot
+
+# Needed for using haptics devices
```

Comment:
please add what is the difference between https://github.com/ykawamura96/Geomagic_Touch_ROS_Drivers and https://github.com/bharatm11/Geomagic_Touch_ROS_Drivers, and list of PRs that should be merged to use https://github.com/bharatm11/Geomagic_Touch_ROS_Drivers

Reply:
Done in #1543 and #1544